weblate_web/models.py

```diff
@@ -634,10 +634,10 @@ def get_suggestions(self):
                     (
                         "backup",
                         _("Backup service"),
-                        _("Easily put your backups in a safe place."),
-                        _("Encrypted and automatic backups, always available."),
+                        _("Easily safeguard all work in a safe place."),
+                        _("Encrypted and automatic, always available."),
                         "img/Support-Backup.svg",
-                        _("Backup daily"),
+                        _("Daily backups"),
```

Comment:
This is a button label, so "Backup daily" makes sense. This is how it looks now:

![image](https://user-images.githubusercontent.com/212189/112312778-78db5800-8ca7-11eb-8541-b7f658e30fb4.png)

Reply:
What happens if  you click it?

Reply:
These are suggestions for service upgrades, clicking on the button triggers purchase of the described upgrade.

Reply:
I know what I don't like about it. "Backup daily" should be "Backup, daily", or "Back up daily" ~ on a daily basis.

Reply:
good point, *backup* is a noun, the button should use a verb which is *back up*